### ISSUE TITLE
OBO STRUCTURE ERROR not specific enough

### ISSUE THREAD

**Issue by cmungall** [ontodev/robot#284]
```
[Term]
id: X:1
name: n1
name: n2
```

```
$ robot convert -i foo.obo -o foo2.obo
OBO STRUCTURE ERROR the ontology does not conform to OBO structure rules
For details see: http://robot.obolibrary.org/convert#obo-structure-error

usage: robot convert --input <file> --format <format> --output <file>
```

In contrast owltools gives:

```
Exception in thread "main" org.semanticweb.owlapi.model.OWLOntologyStorageException: org.obolibrary.oboformat.model.FrameStructureException: multiple name tags not allowed. in frame:Frame(X:1 id( X:1)name( n1)name( n2))
        at org.semanticweb.owlapi.oboformat.OBOFormatRenderer.render(OBOFormatRenderer.java:90)
        at org.semanticweb.owlapi.oboformat.OBOFormatStorer.storeOntology(OBOFormatStorer.java:42)
        at org.semanticweb.owlapi.util.AbstractOWLStorer.storeOntology(AbstractOWLStorer.java:155)
        at org.semanticweb.owlapi.util.AbstractOWLStorer.storeOntology(AbstractOWLStorer.java:119)
        at uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.saveOntology(OWLOntologyManagerImpl.java:1366)
        at uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.saveOntology(OWLOntologyManagerImpl.java:1343)
        at owltools.io.ParserWrapper.saveOWL(ParserWrapper.java:289)
        at owltools.io.ParserWrapper.saveOWL(ParserWrapper.java:209)
        at owltools.cli.CommandRunner.runSingleIteration(CommandRunner.java:3779)
        at owltools.cli.CommandRunnerBase.run(CommandRunnerBase.java:76)
        at owltools.cli.CommandRunnerBase.run(CommandRunnerBase.java:68)
        at owltools.cli.CommandLineInterface.main(CommandLineInterface.java:12)
Caused by: org.obolibrary.oboformat.model.FrameStructureException: multiple name tags not allowed. in frame:Frame(X:1 id( X:1)name( n1)name( n2))
        at org.obolibrary.oboformat.model.Frame.checkMaxOneCardinality(Frame.java:424)
        at org.obolibrary.oboformat.model.Frame.check(Frame.java:405)
        at org.obolibrary.oboformat.model.OBODoc.check(OBODoc.java:390)
        at org.obolibrary.oboformat.writer.OBOFormatWriter.write(OBOFormatWriter.java:189)
        at org.semanticweb.owlapi.oboformat.OBOFormatRenderer.render(OBOFormatRenderer.java:88)
        ... 11 more
```

which is ugly and opaque but a naive user at least has a clue..

**Comment by cmungall** [comment]
once #267 is done we can also suggest to the user that they run `report`, as the checks should subsume the obo structure checks (except for maybe some obo-specific things like comment = cardinality of 1)

**Comment by rctauber** [comment]
I could add the message after the `OBO STRUCTURE ERROR`, e.g.

![image](https://user-images.githubusercontent.com/16750150/39263882-73fcb408-4891-11e8-9a5d-56596b1176ce.png)

**Comment by rctauber** [comment]
Do you think the first 100 characters is sufficient for the main error message (see below)? Users can run the command again with `-vvv` to see the full message.
```
OBO STRUCTURE ERROR Ontology does not conform to OBO structure rules:
	multiple def tags not allowed. in frame:Frame(BFO:0000031 id( BFO:0000031)property_value( BFO:000017...
For details see: http://robot.obolibrary.org/errors#obo-structure-error
Use the -vvv option to show the stack trace.
Use the --help option to see usage information.
```

Or should we include the full message like my first example?

**Comment by cmungall** [comment]
I would tend to report everything just in case

**Comment by beckyjackson** [comment]
This is fixed by #453. Just reopen if it needs more discussion 🙂
